libs/common/src/constants/geospatial.constant.ts

```diff
@@ -0,0 +1,6 @@
+export const ADDRESSES = {
+  EXAMPLES: {
+    POSTCODE: 'SW1A2HQ',
+    POSTCODE_WITHOUT_SPACE: 'SW1A 2HQ',
```

Comment:
Should this be `POSTCODE_WITH_SPACE`?

Reply:
I think name is fine, I just switched values by mistake. 

I think UK postcode should have space by default.